sw/device/lib/dif/dif_alert_handler.c

```diff
@@ -992,5 +880,7 @@ dif_result_t dif_alert_handler_get_class_state(
   return kDifOk;
 }
 
+#undef LOC_ALERT_CAUSE_REGS_CASE_
```

Comment:
I'd like a lint to make sure that all X macros are un'defed if we care... but if not, we shouldn't be bothering. I suspect it's not worth it because this is a .c file, not a header.